scripts/run_e2e_tests.py

```diff
@@ -446,18 +446,41 @@ def get_chrome_driver_version():
     This method follows the steps mentioned here:
     https://chromedriver.chromium.org/downloads/version-selection
     """
-    try:
-        proc = subprocess.Popen(
-            ['google-chrome', '--version'], stdout=subprocess.PIPE)
-        output = proc.stdout.readline()
-    except OSError:
-        raise Exception(
-            'Failed to execute "google-chrome --version" command. This is '
-            'used to determine the chromedriver version to use. Please set '
-            'the chromedriver version manually using --chrome_driver_version '
-            'flag. To determine the chromedriver version to be used, please '
-            'follow the instructions mentioned in the following URL:\n'
-            'https://chromedriver.chromium.org/downloads/version-selection')
+    if common.is_mac_os():
+        try:
+            # Since the command below includes file names that have spaces,
```

Comment:
Add links to online references for each of these points.

Reply:
Don't need them anymore :) but thanks, I will do this in the future if needed.